fieldmaptrack/fieldmap.py

```diff
@@ -155,9 +155,10 @@ def __init__(self,
         # spacing between consecutive coordinate values:
         self.rx_step, self.step, self.rz_step = None, None, None
 
-        if self.field_function is None and self.filename is None:
+        options = {self.field_function, self.filename, content}
+        if len(options) == 1 and None in options:
             raise IrregularFieldMap('source of field not defined!')
-        if self.field_function is not None and self.filename is not None:
+        if len(options) > 2 or None not in options:
```

Comment:
Didn't exactly understand these checks. Prevous checks were more intuitive.

Reply:
I agree, @lucashfrancisco ! but now that we have 3 different options of passing the fieldmap this was the first implementation that was both correct and concise. but maybe a more verbose documentation is better for readability!

Reply:
@lucashfrancisco , i rewrote argument options. I think the code is easier to read now. take a look.

Reply:
@xresende, this looks great now :)